src/plotApplications/components/infoCheck/PlotApplicationTargetInfoCheck.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React, {PureComponent} from 'react';
+import PlotApplicationInfoCheckCollapse from "./PlotApplicationInfoCheckCollapse";
+
+type Props = {
+
+};
+
+type State = {
+
+};
+
+class PlotApplicationTargetInfoCheck extends PureComponent<Props, State> {
+  render(): React$Node {
+    return (
+      <PlotApplicationInfoCheckCollapse className="collapse__secondary" headerTitle="Kohteen käsittelytiedot">
+
```

Comment:
Does this need the closing tag since component does not have any children?

Reply:
Not in its current state indeed, being largely just a placeholder for future development (I believe target info checks aren't even in the BE yet).